pkg/koordlet/runtimehooks/hooks/groupidentity/interceptor.go

```diff
@@ -35,6 +60,27 @@ func (b *bvtPlugin) SetPodBvtValue(p protocol.HooksProtocol) error {
 	podQOS := ext.GetQoSClassByAttrs(req.Labels, req.Annotations)
 	podKubeQOS := util.GetKubeQoSByCgroupParent(req.CgroupParent)
 	podBvt := r.getPodBvtValue(podQOS, podKubeQOS)
+
+	// pod annotatations take precedence to the default CPUQoS
+	cfg, err := getPodCPUQoSConfig(req.Annotations)
```

Comment:
better to use DefaultCPUQOS() as default and merge with annotation, besides, cfg.Enable == nil means disabled by default